### ISSUE TITLE
upload.sh produces a curl error

### ISSUE THREAD

**Issue by squell** [squell/bb-scripts#49]
On lilo5 and lilo4:
```
$ ./upload.sh grades.csv
Still logged in!
curl: option --form: is badly used here
curl: try 'curl --help' or 'curl --manual' for more information
Please set the BlackBoard interface language to English.
```

**Comment by squell** [comment]
Ok, the problem here is that the function nonce() no longer finds the correct nonce, it seems.

**Comment by squell** [comment]
False alarm, sorry.

**Comment by squell** [comment]
Fixed in 4fe55c9, by the way.
